extensions/vertx-http/deployment/src/main/resources/dev-templates/main.html

```diff
@@ -25,6 +25,8 @@
   </title>
  </head>
  <body>
+  <script src="{frameworkRootPath}/dev/resources/js/jquery-3.5.1.min.js"></script>
```

Comment:
Not sure about this change, I just need the `jQuery` to be imported before my body. I don't know if something else could be done.